lib/smart_answer/calculators/minimum_wage_calculator.rb

```diff
@@ -208,5 +208,13 @@ def charged_accomodation_adjustment(charge, number_of_nights)
     def self.historical_minimum_wage_data
       @historical_minimum_wage_data ||= YAML.load_file(Rails.root.join("lib/data/minimum_wage_data.yml"))[:minimum_wage_data]
     end
+
+    def date_from_environment_variable
```

Comment:
@leenagupte `MinimumWageCalculator` doesn't use `RatesQuery` (in fact, it pre-dates it by about 3 years).

They do different things as well: this function returns `Date.now` if `RATES_QUERY_DATE` is unset, whereas the implementation in `RatesQuery` returns `nil` in that situation.